src/video_core/vulkan_common/vulkan_device.cpp

```diff
@@ -231,9 +231,14 @@ std::vector<std::string> GetSupportedExtensions(vk::PhysicalDevice physical) {
     return supported_extensions;
 }
 
+bool IsExtensionSupported(std::span<const std::string> supported_extensions,
+                          const std::string& extension) {
```

Comment:
```suggestion
                          std::string_view extension) {
```

should be able to work here iirc